File: jcarrollonlinev4.backend/Models/Users/UserDetailModel.cs
namespace jcarrollonlinev4.backend.Models.Users
{
    public class UserDetailModel : UserModelBase
    {
        //private static Logger logger = LogManager.GetCurrentClassLogger();

        public UserDetailModel()
        {
            UserInfoModel = new UserItemModel();
            UserStatsModel = new UserStatsModel();
        }
        public UserItemModel UserInfoModel { get; set; }
        public UserStatsModel UserStatsModel { get; set; }
    }
}

File: jcarrollonlinev4.frontend/src/models/Users/UserDetailModel.ts
import { UserInfoModel } from "../user-info";
import { UserModelBase } from "./UserModelBase";

    export interface UserDetailModel extends UserModelBase
    {
        //private static Logger logger = LogManager.GetCurrentClassLogger();
      UserItemModel?: UserInfoModel;
      UserStatsModel?: UserStatsModel;
    }
}
